src/ORM/DataObject.php

```diff
@@ -444,27 +520,109 @@ protected function duplicateManyManyRelations($sourceObject, $destinationObject,
     }
 
     /**
-     * Duplicates a single many_many relation from one object to another
+     * Duplicates a single many_many relation from one object to another.
      *
      * @param DataObject $sourceObject
      * @param DataObject $destinationObject
-     * @param string $manyManyName
+     * @param string $relation
      */
-    protected function duplicateManyManyRelation($sourceObject, $destinationObject, $manyManyName)
+    protected function duplicateManyManyRelation($sourceObject, $destinationObject, $relation)
     {
         // Ensure this component exists on the destination side as well
-        if (!static::getSchema()->manyManyComponent(get_class($destinationObject), $manyManyName)) {
+        if (!static::getSchema()->manyManyComponent(get_class($destinationObject), $relation)) {
```

Comment:
Shouldn't this throw an exception?

Reply:
Good idea, I'd just copied it but let's be more strict. :)